src/mutex/spin.rs

```diff
@@ -201,6 +201,17 @@ impl<T: ?Sized, R> SpinMutex<T, R> {
         self.lock.load(Ordering::Relaxed)
     }
 
+    /// Returns `true` if the lock is currently held.
+    ///
+    /// # Safety
+    ///
+    /// This function checks whether the mutex is locked using an acquire
+    /// ordering.
+    #[inline(always)]
+    pub fn is_locked_acquire(&self) -> bool {
+        self.lock.load(Ordering::Relaxed)
```

Comment:
did you intend for this to be `Ordering::Acquire`? I assume so based on the comments and the other similar functions you added.